ci-operator/step-registry/upi/libvirt/conf/upi-libvirt-conf-commands.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+echo "Installing from initial release ${RELEASE_IMAGE_LATEST}"
+
+openshift-install version
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+CLUSTER_NAME="libvirt-s390x-amd64-0-0"
+BASE_DOMAIN="ci"
+
+cat >> "${CONFIG}" << EOF
+apiVersion: v1
+baseDomain: ${BASE_DOMAIN}
+metadata:
+  name: ${CLUSTER_NAME}
+controlPlane:
+  architecture: ${ARCH}
+  hyperthreading: Enabled
+  name: master
+  replicas: ${MASTER_REPLICAS}
+compute:
+- architecture: ${ARCH}
+  hyperthreading: Enabled
+  name: worker
+  replicas: 0
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+fips: false
+pullSecret: >
+  $(<"${CLUSTER_PROFILE_DIR}/pull-secret")
```

Comment:
Use `yq` instead to make sure it never leaks:
```
yq e ".pullSecret = load_str(${CLUSTER_PROFILE_DIR}/pull-secret)" -i ${CONFIG}/install-config.yaml
```

Reply:
This doesn't work with the old version of yq, as far as I understand. I've pulled in a newer version for now in the conf step itself (as I did in the actual installation steps), but I think it should at some point move that to the libvirt image we're using for this step. (https://github.com/openshift/installer/blob/master/images/libvirt/Dockerfile.ci#L35-L40)